packages/bpk-component-dialog/stories.js

```diff
@@ -0,0 +1,134 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2017 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { cssModules, withDefaultProps } from 'bpk-react-utils';
+
+import BpkButton from 'bpk-component-button';
+import BpkText from 'bpk-component-text';
+
+import BpkDialog from './index';
+
+import STYLES from './stories.scss';
+
+const getClassName = cssModules(STYLES);
+
+const Paragraph = withDefaultProps(BpkText, {
+  textStyle: 'base',
+  tagName: 'p',
+  className: getClassName('bpk-dialog-paragraph'),
+});
+
+class DialogContainer extends Component {
+  constructor() {
+    super();
+
+    this.onOpen = this.onOpen.bind(this);
+    this.onClose = this.onClose.bind(this);
+
+    this.state = {
+      isOpen: false,
+    };
+  }
+
+  onOpen() {
+    this.setState({
+      isOpen: true,
+    });
+  }
+
+  onClose() {
+    this.setState({
+      isOpen: false,
+    });
+  }
+
+  render() {
+    const { buttonText, ...rest } = this.props;
+
+    return (
+      <div>
+        <div id="application-container">
```

Comment:
This should match the modal stories where by we specify a custom render target, see https://github.com/Skyscanner/backpack/blob/master/packages/bpk-component-modal/stories.js#L68.

This is for theming btw ;)